diff --git a/conf/config.py b/conf/config.py
index 2f056f8..2a91236 100644
--- a/server/conf/config.py
+++ b/server/conf/config.py
@@ -52,7 +52,7 @@ class BaseConfiguration(object):
     # noauth: no authentication is enabled. Useful for development particularly.
     # kerberos: Kerberos authentication is enabled.
     # openidc: OpenIDC authentication is enabled.
-    AUTH_BACKEND = ''
+    AUTH_BACKEND = 'openidc'
 
     # Used for Kerberos authentication and to query user's groups.
     # Format: ldap://hostname[:port]
